source/brailleDisplayDrivers/alva.py

```diff
@@ -258,44 +272,59 @@ def _handleInput(self, group, number):
 			# This begins a new key combination.
 			self._ignoreKeyReleases = False
 
-	def _hidDisplay(self, cells):
-		for offset in range(0, len(cells), ALVA_BRAILLE_OUTPUT_MAX_SIZE):
-			cellsToWrite = cells[offset:offset+ALVA_BRAILLE_OUTPUT_MAX_SIZE]
-			self._dev.write("{id}{offset}{count}{cells}".format(
-				id=ALVA_BRAILLE_OUTPUT_REPORT,
-				offset=chr(offset),
-				count=chr(len(cellsToWrite)),
-				cells=cellsToWrite
-			))
-
-	def _ser6Display(self, cells):
-		self._ser6SendMessage(b"B", chr(0)+chr(len(cells))+cells)
-
-	def display(self, cells):
+	def _hidDisplay(self, cellBytes: bytes) -> None:
+		for offset in range(0, len(cellBytes), ALVA_BRAILLE_OUTPUT_MAX_SIZE):
+			cellsToWrite = cellBytes[offset:offset+ALVA_BRAILLE_OUTPUT_MAX_SIZE]
+			data = b"".join([
+				ALVA_BRAILLE_OUTPUT_REPORT,
+				intToByte(offset),

```

Comment:
I believe there is a [proposal](https://www.python.org/dev/peps/pep-0467/) planned for python 3.9 for this to be called bchr. We could anticipate for this.

Reply:
I don't think we should name this `bchr` if that is what you are suggesting, that could lead to an unexpected change in behaviour when we upgrade.